exporter/otlp-grpc/opentelemetry-exporter-otlp-grpc.gemspec

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'opentelemetry/exporter/otlp/grpc/version'
+
+Gem::Specification.new do |spec|
+  spec.name        = 'opentelemetry-exporter-otlp-grpc'
+  spec.version     = OpenTelemetry::Exporter::OTLP::GRPC::VERSION
+  spec.authors     = ['OpenTelemetry Authors']
+  spec.email       = ['cncf-opentelemetry-contributors@lists.cncf.io']
+
+  spec.summary     = 'OTLP exporter for the OpenTelemetry framework'
+  spec.description = 'OTLP exporter for the OpenTelemetry framework'
+  spec.homepage    = 'https://github.com/open-telemetry/opentelemetry-ruby'
+  spec.license     = 'Apache-2.0'
+
+  spec.files = ::Dir.glob('lib/**/*.rb') +
+               ::Dir.glob('*.md') +
+               ['LICENSE', '.yardopts']
+  spec.require_paths = ['lib']
+  spec.required_ruby_version = '>= 2.5.0'
+
+  spec.add_dependency 'grpc'
```

Comment:
Do we have a min supported version or are we open to anything?

Reply:
Leaving this open to whoever picks this up.